AWESOME_RGF.md

```diff
@@ -0,0 +1,26 @@
+Awesome RGF
+===========
+This page contains a curated list of blogs, competitions solutions about RGF.
+It is inspired by [awesome-xgboost](https://github.com/dmlc/xgboost/blob/master/demo/README.md),
+
+Please feel free to send a pull request if you find things that belongs to here.
+
+Contents
+--------
+- [Blog Post](#blog-post)
+- [Machine Learning Competition Winning Solutions](#machine-learning-competition-winning-solutions)
+
+### Blog Post
+
+* [Regularized Greedy Forest (RGF) - Nice alternative to tree-boosting](https://www.linkedin.com/pulse/regularized-greedy-forest-rgf-nice-alternative-marios-michailidis/)
+* [An Introductory Guide to Regularized Greedy Forests (RGF) with a case study in Python](https://www.analyticsvidhya.com/blog/2018/02/introductory-guide-regularized-greedy-forests-rgf-python/)
+* [Singularity as a software distribution / deployment tool](http://mlampros.github.io/2018/07/26/singularity_containers/)
```

Comment:
I believe that this link is better place to the R-package README - there is no RGF-specific information, just general words about Singularity...

Reply:
OK, I deleted R specific links.
These links could be appended to R-package in the future.